crt_portal/cts_forms/views.py

```diff
@@ -340,6 +350,7 @@ def serialize_data(report, request, report_id):
 class ProfileView(LoginRequiredMixin, FormView):
     """Can be used for updating section filter for a profile"""
     form_class = ProfileForm
+    template_nmae = 'forms/complaint_view/index/index.html'
```

Comment:
Typo here, but it also looks like this view doesn't render any templates. Is this necessary?